core/src/main/java/com/linecorp/armeria/internal/server/annotation/ResponseConverterFunctionSelector.java

```diff
@@ -0,0 +1,186 @@
+/*
+Copyright 2022 LINE Corporation
+
```

Comment:
Could you fix the copyright?
```suggestion
/*
 * Copyright 2022 LINE Corporation
 *
```

Reply:
Fixed in a18d9cb5eeb77c0508db194e76e052a312d87c36